choreolib/src/main/java/choreo/auto/AutoChooser.java

```diff
@@ -92,12 +102,29 @@ public void update() {
   /**
    * Add an auto routine to the chooser.
    *
-   * <p>An auto routine is a function that takes an AutoFactory and returns a Command. These
-   * functions can be static, a lambda or belong to a local variable.
+   * <p>An auto routine is a function that takes an {@link AutoFactory} and returns a {@link
```

Comment:
I don’t like the naming here, is an auto routine a function or that function’s return?

Also, should state that the function is executed when the chooser is changed to the corresponding value.

Reply:
The docs state in multiple places that the function is only executed when the chooser is changed.
I agree I have to fix the wording.